packages/fxa-settings/src/components/Settings/PageSecurityEvents/SecurityEvent.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+export function SecurityEvent({
+  name,
+  createdAt,
+  verified,
+}: {
+  name: string;
+  createdAt: number;
+  verified?: boolean;
+}) {
+  const createdAtDateText = Intl.DateTimeFormat('en-US', {
+    year: 'numeric',
+    month: 'numeric',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  }).format(new Date(createdAt));
+
+  const formattedName = `security-events-${name.split('.').join('-')}`;
```

Comment:
nit: should we use `.replace()` w/ regexp or `.replaceAll()` here? Doubt that it's any better/faster than a split+join, but :shrug:

```js
const name = "peter.is.a.weirdo";
console.log({ name }); // {name: 'peter.is.a.weirdo'}
console.log(name.replace(/\./g, "-")); // peter-is-a-weirdo
console.log(name.replaceAll(".", "-")); // peter-is-a-weirdo
```